README.md

```diff
@@ -46,6 +46,48 @@ path.defaults.width(300).to_url # Resets parameters
 path.rect(x: 0, y: 50, width: 200, height: 300).to_url # Rect helper
 ```
 
+## Srcset Generation
+
+The imgix gem allows for generation of custom `srcset` attributes, which can be invoked through `Imgix::Path#to_srcset`. By default, the `srcset` generated will allow for responsive size switching by building a menu of image-width mappings.
```

Comment:
I like "choosing from a menu" as a metaphor for the browser's `sizes`/`srcset` interaction, but I think "list" is a better word for this context.